common/reviews/api/rush-lib.api.ts

```diff
@@ -1,3 +1,8 @@
+// @internal (undocumented)
+class _CLI {
```

Comment:
>_CLI [](start = 6, length = 4)

We decided to rename this to "Rush" since Rush.version will go here.

For future reference, acronyms (with 3 letters or more) should use CamelCase